src/DoctrineAuditBundle/DependencyInjection/Configuration.php

```diff
@@ -25,6 +25,10 @@ public function getConfigTreeBuilder()
                     ->defaultTrue()
                 ->end()
 
+                ->scalarNode('storage_entity_manager')
+                    ->cannotBeEmpty()
```

Comment:
It should not be null by default to conserver the default configuration ?

Reply:
2 schools here:
1. the option can be undefined without default value => we have to check if the config option is set afterwards
2. the option cannot be undefined because of a default (null) value => we have to check if the config option is not null afterwards

Both do the same with different approach but I'd say the 2nd way makes the configuration structure more consistent => I go for this one.